src/Language/Fixpoint/Types/Config.hs

```diff
@@ -250,7 +251,7 @@ defConfig = Config {
   , rewriteAxioms            = False &= help "allow axiom instantiation via rewriting (PLE)"
   , pleWithUndecidedGuards   =
       False
-        &= name "ple-with-undecided-guards"
+        &= name "plewithundecidedguards"
```

Comment:
Do you mean that keeping dashes here caused CI to fail? It would be helpful to see what the error is.

Reply:
Yes keeping the dashes there made the intId test fail (locally) as CmdArgs kept saying “unknown flag”. More mysteriously, the test worked if I just wrote “—ple”…

Reply:
I think it may be some strange MacOS thing ... have reverted the config flag name, will leave it be if it works in the CI!